### PR TITLE
PST-07 UI - Add Profile Pic avatar along with dropdown arrow. Dropdown having profile and logout

### PR DESCRIPTION
**Pull request by suneel-katta**
PST-07 UI - Add Profile Pic avatar along with dropdown arrow. Dropdown having profile and logout

### CODE REVIEW

File: src/main/webapp/student_navigation_bar.html [hunk 1]
```diff
@@ -36,10 +33,24 @@
             <li><a class="dropdown-item text-white" href="#">Something else here</a></li>
           </ul>
         </li>
-        <li class="nav-item" style="float:right">
-          <a class="nav-link text-white" href="./home.jsp">Logout</a>
-        </li>
       </ul>
+
```
Comment (phanisofttech) [inline src/main/webapp/student_navigation_bar.html:37]: Remove extra lines only keep one lilne

File: src/main/webapp/student_navigation_bar.html [hunk 2]
```diff
@@ -36,10 +33,24 @@
             <li><a class="dropdown-item text-white" href="#">Something else here</a></li>
           </ul>
         </li>
-        <li class="nav-item" style="float:right">
-          <a class="nav-link text-white" href="./home.jsp">Logout</a>
-        </li>
       </ul>
+      
+
+
```
Comment (phanisofttech) [inline src/main/webapp/student_navigation_bar.html]: Add comment with description

File: src/main/webapp/student_navigation_bar.html [hunk 3]
```diff
@@ -36,10 +33,24 @@
...
+			aria-expanded="false">
+			<img src="https://tse4.mm.bing.net/th?id=OIP.d9HP9ELDsCdvaqsRxqQ1BwHaIS&pid=Api&P=0&h=180" alt="Logo" width="40"
+				height="40" class="d-inline-block align-top">
+		</button>
+		<ul class="dropdown-menu dropdown-menu-end" aria-labelledby="dropdownMenuButton">
+			<li><a class="dropdown-item" href="#">Profile</a></li>
+			<li><a class="dropdown-item" href="#">Logout</a></li>
+		</ul>
+	</div>
+
```
Comment (phanisofttech) [inline src/main/webapp/student_navigation_bar.html:50]: Remove extra spaces only keep one space

**Review by phanisofttech — CHANGES_REQUESTED** [review]

**Comment by suneel-katta** [comment]
Changes altered @phanisofttech

File: src/main/webapp/student_navigation_bar.html (hunk 1 above)
Comment (suneel-katta) [reply src/main/webapp/student_navigation_bar.html:37]: altered

File: src/main/webapp/student_navigation_bar.html (hunk 2 above)
Comment (suneel-katta) [reply src/main/webapp/student_navigation_bar.html]: altered

File: src/main/webapp/student_navigation_bar.html (hunk 3 above)
Comment (suneel-katta) [reply src/main/webapp/student_navigation_bar.html:50]: altered

**Pull request merged**